src/mscorlib/corefx/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -820,6 +820,10 @@ internal String DateSeparator
                 Contract.Assert(this.dateSeparator != null, "DateTimeFormatInfo.DateSeparator, dateSeparator != null");
                 return (this.dateSeparator);
             }
+            set
+            {
+                throw null;
```

Comment:
We should throw PlatformNotSupportedException, until we fill in this implementation.